aspnetcore/test/troubleshoot.md

```diff
@@ -59,16 +59,4 @@ In the **New Project** dialog for ASP.NET Core, you may see the following warnin
 This warning appears when the environment variable `PATH` doesn't point to any .NET Core SDKs on the machine. To resolve this problem:
 
 * Install or verify the .NET Core SDK is installed.
-* Verify the `PATH` environment variable points to the location the SDK is installed. The installer normally sets the `PATH`.
-
-::: moniker range=">= aspnetcore-2.1"
-
-### Use of IHtmlHelper.Partial may result in app deadlocks
-
-In ASP.NET Core 2.1 and later, calling `Html.Partial` results in an analyzer warning due to the potential for deadlocks. The warning message is:
-
-> Use of IHtmlHelper.Partial may result in application deadlocks. Consider using `<partial>` Tag Helper or `IHtmlHelper.PartialAsync`.
-
-Calls to `@Html.Partial` should be replaced by `@await Html.PartialAsync` or the partial tag helper `<partial name="_Partial" />`.
-
-::: moniker-end
+* Verify the `PATH` environment variable points to the location the SDK is installed. The installer normally sets the `PATH`.
```

Comment:
You can introduce the verb's object with "that." It might read a hair smoother.

> Verify that the \`PATH\` environment variable points to the location the SDK is installed.